boot/bootutil/src/loader.c

```diff
@@ -1902,6 +1902,43 @@ boot_update_hw_rollback_protection(struct boot_loader_state *state)
 #endif
 }
 
+static int
+check_downgrade_prevention(struct boot_loader_state *state)
+{
+#if defined(MCUBOOT_DOWNGRADE_PREVENTION) && (defined(MCUBOOT_SWAP_USING_MOVE) || defined(MCUBOOT_SWAP_USING_SCRATCH))
+    uint32_t security_counter[2];
+    int rc;
+
+    if (MCUBOOT_DOWNGRADE_PREVENTION_SECURITY_COUNTER) {
+        /* If there was security no counter in slot 0, allow swap */
+        rc = bootutil_get_img_security_cnt(&(BOOT_IMG(state, 0).hdr), BOOT_IMG(state, 0).area, &security_counter[0]);
```

Comment:
Please try to keep lines to 80-90 characters. These lines, including the #if`` above should be split across two or more lines.

Reply:
lines length limited as requested